docs/platforms/kotlin-multiplatform/features/index.mdx

```diff
@@ -0,0 +1,32 @@
+---
+title: Features
+sidebar_order: 1
+description: "Learn about the features of Sentry's Kotlin Multiplatform SDK."
+---
+
+Sentry's Kotlin Multiplatform SDK enables automatic reporting of errors and exception.
+The SDK provides a unified API for all supported platforms, directly leveraging the native SDKs for each platform.
+Therefore it also supports the same features as the native SDKs.
+
+- [Apple features](/platforms/apple/features)
+- [Android features](/platforms/android/features)
+- [Java features](/platforms/java/features)
```

Comment:
```suggestion
Sentry's Kotlin Multiplatform SDK enables automatic reporting of errors and exceptions.
Because the SDK provides a unified API for all supported platforms directly leveraging the native SDKs for each platform, it also supports the same features as the native SDKs. Click on a link below to see relevant features for your platform:

- [Apple features](/platforms/apple/features)
- [Android features](/platforms/android/features)
- [Java features](/platforms/java/features)
```